pkg/exchange/okex/convert.go

```diff
@@ -55,11 +55,11 @@ func toGlobalBalance(account *okexapi.Account) types.BalanceMap {
 	return balanceMap
 }
 
-type WebsocketSubscription struct {
-	Channel        string `json:"channel"`
-	InstrumentID   string `json:"instId,omitempty"`
-	InstrumentType string `json:"instType,omitempty"`
-}
+// type WebsocketSubscription struct {
```

Comment:
can you remove the redundant code instead of comment? This can help us to review the code

Reply:
already rm in the latest commit.